packages/ui/nuxt.config.ts

```diff
@@ -129,7 +129,7 @@ export default defineNuxtConfig({
     defaultLocale: 'en-US',
     detectBrowserLanguage: {
       useCookie: true,
-      cookieKey: 'i18n_redirected',
+      cookieKey: 'i18n_redirected_2',
```

Comment:
why do we need to rename this? Can the cookie contain a value from the old version that isn't valid anymore?